swiftlint.yml

```diff
@@ -56,7 +56,9 @@ line_length:
 force_cast: warning
 force_try: warning
 
+sorted_imports: error
```

Comment:
A bit harsh, oh well

Reply:
it will be autocorrected before an error is thrown 😄

Arnas asked for this so I'm enforcing it the hard way